articles/quickstart.md

```diff
@@ -96,26 +108,24 @@ Our goal is to prepare two qubits in a [Bell state](https://en.wikipedia.org/wik
     }
     ```
 
-    This operation may now be called to set a qubit to a known state (`Zero` or `One`).
+    This operation may now be called to set a qubit to a classical state, either returning `Zero` 100% of the time or returning `One` 100% of the time.  `Zero` and `One` are constants that represent the only two possible results of a measurement of a qubit.
 
-    We measure the qubit, if it's in the state we want, we leave it alone, otherwise, we flip it with the `X` gate.
+    The operation `Set` measures the qubit, if it's in the state we want, we leave it alone, otherwise, by executing the `X` operation, we change the qubit state to the desired state.
```

Comment:
This sentence feels a bit awkward at the moment. I apologize that I can't quite put my finger on why, though.

Reply:
Don't want to say "flip", because new learner does not have conceptual image of qubit where "flipping" is understood.

Reply:
It works for me, I agree it's more intuitive than "flip"

Reply:
To be sure, wasn't suggestion going back to flip. Maybe it would help to separate into two sentences?
```suggestion
    The operation `Set` measures the qubit.
    If the qubit is in the state we want, `Set` leaves it alone; otherwise, by executing the `X` operation, we change the qubit state to the desired state.
```